include/triton/common/triton_json.h

```diff
@@ -184,6 +184,12 @@ class TritonJson {
             std::string(GetParseError_En(document_.GetParseError())) + " at " +
             std::to_string(document_.GetErrorOffset())));
       }
+      TRITONJSON_STATUSTYPE status =
+          ParseErrorHandler(document_, std::string(base, size));
```

Comment:
Can you add a before/after example in the PR description to highlight the new error/UX improvement?

Reply:
Sure! Let me run it through CI then. I wanted your feedback first in case there was a reason not to do this.